docs/lcdproc-user/drivers/futaba.docbook

```diff
@@ -0,0 +1,140 @@
+        </listitem>
+
+        <listitem>
+          <para>The patched version of
+          <application>scripts.xbmc.lcproc</application> can be found <ulink
+          url="https://github.com/ajw107/script.xbmc.lcdproc">here</ulink>,
+          this should be added upstream soon too.</para>
+        </listitem>
+      </itemizedlist>
+    </note>
+  </sect2>
+
+  <sect2 id="futaba-requirements">
+    <title>Requirements</title>
+
+    <para>The driver is based on the <ulink
+    url="http://libusb.info/"><filename>libusb</filename></ulink> USB library
+    (Versions 1.0 and 0.1), which should make it work with Linux, the
+    different BSB variants as well as Darwin/MacOS X.</para>
+
+    <note>
+      <para>When using a <filename>libusb</filename> based driver like
+      <code>futaba</code>, <application>LCDd</application> needs to be started
+      as root.</para>
+    </note>
+
+    <para>On Linux, the only kernel module required is the USB host controller
+    driver (<filename>uhci_hcd</filename> on the M300) to fire up the USB bus
+    to which the LCD is attached. For other operating systems, analogous
+    requirements apply.</para>
+
+    <para>Lastly, for <filename>libusb</filename> to work correctly, the
+    <filename>usbfs</filename> file system must be mounted on
+    <filename>/proc/bus/usb</filename>, e.g. using the command <code>mount -t
+    usbfs usbfs /proc/bus/usb</code> or by your system's default
+    configuration.</para>
+  </sect2>
+
+  <sect2 id="futaba-config">
+    <title>Configuration in LCDd.conf</title>
+
+    <sect3 id="futaba-config-section">
+      <title>[futaba]</title>
+
+      <variablelist>
+
+        <varlistentry>
+          <term><property>Size</property> =
+          <parameter>WIDTHxHEIGHT</parameter></term>
+
+          <listitem>
+            <para>Should not be changed from the default of 7x1.</para>
+          </listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term><property>TitleSpeed</property> =
+          <parameter><replaceable>TITLESPEED</replaceable></parameter></term>
+
+          <listitem>
+            <para>Set the speed the text marquee scrolls at. Legal values are:
+            <literal>1</literal> - <literal>10</literal>. If not given, it
+            defaults to <literal>5</literal>.</para>
+          </listitem>
+        </varlistentry>
+      </variablelist>
+
+<note>
+<para>There are no Brightness, Contrast or Back-light controls as
+the display does not support these functions</para>
+</note>
+
+    </sect3>
+  </sect2>
+
+  <sect2 id="futaba-status">
+    <title>futaba driver status</title>
+
+    <para>All icons/indicators function well. Currently the ':' (colon) or '.'
+    (dot) in between characters do NOT operate. In a future update I hope to
+    get these to work, with three settings: one to enable/disable flashing;
+    one to set the flash rate; and one to choose between ':' (colon) or '.'
+    (dot).</para>
+  </sect2>
```

Comment:
I have a smilar situation in the olimex_1x9 driver. Maybe you can take some inspiration from there or maybe you come up with something better, for me to copy ... ;)